presto-docs/src/main/sphinx/functions/conversion.rst

```diff
@@ -22,6 +22,35 @@ Conversion Functions
 
     Like :func:`cast`, but returns null if the cast fails.
 
+Data Size
+---------
+
+The ``parse_data_size`` function supports the following units:
+
+======= ============= ========
```

Comment:
Let's use superscript for the values:
```
======= ============= ==============
Unit    Description   Value
======= ============= ==============
``B``   Bytes         1
``kB``  Kilobytes     1024
``MB``  Megabytes     1024\ :sup:`2`
``GB``  Gigabytes     1024\ :sup:`3`
``TB``  Terabytes     1024\ :sup:`4`
``PB``  Petabytes     1024\ :sup:`5`
``EB``  Exabytes      1024\ :sup:`6`
``ZB``  Zettabytes    1024\ :sup:`7`
``YB``  Yottabytes    1024\ :sup:`8`
======= ============= ==============
```
I also fixed `KB` to be `kB`

Reply:
<img width="1071" alt="screen shot 2018-11-06 at 3 14 51 pm" src="https://user-images.githubusercontent.com/9230/48100083-bf16e600-e1d6-11e8-9846-3092bad4844f.png">

Reply:
Cool. I was googling for something like `:sup`. Didn't find it. Thanks.